Sources/SKCore/Configuration.swift

```diff
@@ -0,0 +1,25 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+import Utility
+import PackageModel
```

Comment:
We shouldn't be using `PackageModel` outside of the `SwiftPMWorkspace` module - it's the only place that depends on all of libSwiftPM.  Only things in the `Utility` and `Basic` modules of SwiftPM should be used elsewhere.  Specifically, we could duplicate `BuildConfiguration`, which is a very simple enum, and `BuildFlags` is okay because it's already in `Utility`.